{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-03-26T01:20:00.000+0000",
      "latitude": 51.292316216357385,
      "estimatedRestoration": "2022-03-26T03:00:00.000+0000",
      "message": "Weâ€™re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JQ7098'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JQ7098",
      "reference": "JQ7098",
      "name": "GU51 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GU51 1BU",
        "GU51 1GF",
        "GU51 1GG",
        "GU51 1GH",
        "GU51 1GJ",
        "GU51 1GL",
        "GU51 1GN",
        "GU51 1GP",
        "GU51 1GR",
        "GU51 1GY",
        "GU51 1HA",
        "GU51 1HH",
        "GU51 1HU",
        "GU51 1HX",
        "GU51 1JD",
        "GU51 1JJ",
        "GU51 1JL",
        "GU51 1JP",
        "GU51 1JR",
        "GU51 1JS",
        "GU51 1LG",
        "GU51 1LH",
        "GU51 4PA"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.8471268858406453,
          51.292316216357385
        ],
        "type": "Point"
      },
      "UUID": "42711e2e-3b05-477f-af9b-602a55ac3842",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "fbace26c-39a2-4de1-adbd-a55d359ae5f1"
      ],
      "longitude": -0.8471268858406453,
      "resolved": false
    }
  ]
}
